src/core/domainer.ts

```diff
@@ -6,8 +6,8 @@ module Plottable {
     private _doNice = false;
     private _niceCount: number;
     private _padProportion = 0.0;
-    private _paddingExceptions = new Utils.StrictEqualityAssociativeArray();
-    private _includedValues = new Utils.StrictEqualityAssociativeArray();
+    private _paddingExceptions: Utils.Map<any, any>;
+    private _includedValues: Utils.Map<any, any>;
```

Comment:
`Domainer` as a whole probably needs to take a generic (the `<D>` on a `Scale<D, R>`), but we can put that aside for now.